### PR TITLE
fix(devtools-plugin): enable time-traveling for navigation actions

### PR DESCRIPTION
**Pull request by arturovt**
Issue: #1640

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ $ npm install @ngxs/store@dev
 - Feature: Expose `ActionContext` and `ActionStatus` [#1766](https://github.com/ngxs/store/pull/1766)
 - Feature: `ofAction*` methods should have strong types [#1808](https://github.com/ngxs/store/pull/1808)
 - Feature: Improve contextual type inference for state operators [#1806](https://github.com/ngxs/store/pull/1806)
+- Feature: Devtools Plugin - Enable time-traveling for navigation actions [#1868](https://github.com/ngxs/store/pull/1868)
```
Comment (markwhitfeld) [inline CHANGELOG.md:12]: I think that this is more of a bug fix

File: packages/devtools-plugin/src/devtools.plugin.ts [hunk 2]
```diff
@@ -98,6 +98,15 @@ export class NgxsReduxDevtoolsPlugin implements OnDestroy, NgxsPlugin {
         action.payload.type === ReduxDevtoolsPayloadType.JumpToState
       ) {
         const prevState = JSON.parse(action.state);
+        // This makes friends DevTools and Router plugins with each other.
```
Comment (markwhitfeld) [inline packages/devtools-plugin/src/devtools.plugin.ts:101]:
```suggestion
        // This makes the DevTools and Router plugins friends with each other.
```

**Review by markwhitfeld — APPROVED** [review]
Looking great!
Just a small tweak to wording, and... this is more of a bug fix than a feature in my opinion.

**Comment by arturovt** [comment]
Alright, gonna rename it!

**Pull request merged**